src/controllers/myAPIsController.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-disable no-undef */
+const path = require('path');
+const fs = require('fs');
+const { renderGivenTemplate, loadLayoutFromAPI, renderTemplate } = require('../utils/util');
+const config = require(process.cwd() + '/config.json');
+const constants = require('../utils/constants');
+const adminDao = require('../dao/admin');
+const apiMetadataService = require('../services/apiMetadataService');
+const util = require('../utils/util');
+
+const loadMyAPIs = async (req, res) => {
+    const orgName = req.originalUrl.split("/")[1];
+    const orgId = await adminDao.getOrgId(orgName);
+    const completeTemplatePath = path.join(require.main.filename, '..', 'pages', 'myAPIs', 'page.hbs');
+    const templateResponse = fs.readFileSync(completeTemplatePath, constants.CHARSET_UTF8);
+    const layoutResponse = await loadLayoutFromAPI(orgId);
+    let metaData = await apiMetadataService.getMetadataListFromDB(orgId);
+    const apiRefIds = new Set(metaData.map(api => api.apiReferenceID));
+
+    let subscriptions = [];
+    let applications = [];
+    let subscribedApps = [];
+
+    for (const apiRefId of apiRefIds) {
+        const subs = await loadSubscriptions(res, apiRefId);
+
+        if (subs) {
+            for (const sub of subs.list) {
+                subscriptions.push({
+                    id: sub.subscriptionId,
+                    apiName: sub.apiInfo.name,
+                    applicationName: sub.applicationInfo.name,
+                    applicationId: sub.applicationInfo.applicationId,
+                    throttlingTier: sub.throttlingPolicy,
+                    appStatus: sub.status,
+                });
+
+            }
+        }
+    }
+
+    if (req.query.apiId) {
```

Comment:
lets add a comment explaining the logic in short...

Reply:
Added https://github.com/wso2/api-developer-portal-core/pull/71/commits/13b2241cd5dc9788a322993613ffd06c785c10a2